Server/service/networkService.js

```
@@ -181,13 +210,13 @@ class NetworkService {
 			case this.TYPE_HARDWARE:
 				return await this.requestHardware(job);
+			case this.TYPE_DOCKER:
+				return await this.requestDocker(job);
 			default:
-				this.logger.error({
-					message: `Unsupported type: ${job.data.type}`,
-					service: this.SERVICE_NAME,
-					method: "getStatus",
-				});
-				return {};
+				const err = new Error(`Unsupported type: ${job.data.type}`);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)